tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1090,6 +1090,10 @@ protected:
                (void)expander; // avoid unused variable warnings for older compilers such as gcc 4.9
             }
             trees[slot]->Fill();
+            if (files[slot]->GetFileBytesWritten() > 96L*1024L*1024L) {
```

Comment:
Can we add something like this to the test case? I.e. can we make sure we reproduce the issue we are solving in the testcase, too?